src/main/cliapp/src/service/DataLoadService.js

```diff
@@ -81,6 +81,7 @@ export class DataLoadService extends BaseAuthService {
 		const bulkLoadTypes = {
 			BulkFMSLoad: [
 				'GFF',
+				'HTP_EXPRESSION_DATASET_ANNOTATION',
```

Comment:
I was wrong before about this having to match the backend enum.  It has to match the data type in the FMS, which in this case is 'HTPDATASET'.  This is what gets stored in the `fmsdatatype` column of the `bulkfmsload` table.  In theory this can be different to what is used in the `BackendBulkLoadType` enum (in some cases we've had to do this because the enums don't allow hypens to be used - see the migration v0.32.0.1__gene_molecular_interaction.sql for an example of this), but in this case I would use 'HTPDATASET' for both to avoid confusion.